File: src/main/graphql/io/github/graphql/resolver/DeploymentResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for Deployment
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface DeploymentResolver {

    @Throws(Exception::class)
    fun statuses(deploymentTO: DeploymentTO, after: String?, before: String?, first: Int?, last: Int?): DeploymentStatusConnectionTO?

}

File: src/main/scala-graphql/io/github/graphql/scala/resolver/DeploymentResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._

/**
 * Resolver for Deployment
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait DeploymentResolver {

    @throws[Exception]
    def statuses(deploymentTO: DeploymentTO, after: String, before: String, first: Option[Int], last: Option[Int]): DeploymentStatusConnectionTO

}
